com.unity.netcode.gameobjects/Tests/Editor/com.unity.netcode.editortests.asmdef

```diff
@@ -3,12 +3,23 @@
     "rootNamespace": "Unity.Netcode.EditorTests",
     "references": [
         "Unity.Netcode.Runtime",
-        "Unity.Netcode.Editor"
-    ],
-    "optionalUnityReferences": [
-        "TestAssemblies"
+        "Unity.Netcode.Editor",
+        "UnityEngine.TestRunner",
+        "UnityEditor.TestRunner"
     ],
     "includePlatforms": [
         "Editor"
-    ]
+    ],
+    "excludePlatforms": [],
+    "allowUnsafeCode": true,
```

Comment:
nit: all we need is `"allowUnsafeCode": true` line manually added here (I don't like Unity auto-gen'ing these things all the time, leaving minimal stuff here makes us more future-proof between different Unity Editor versions)